### PR TITLE
feature/added-hi-message

### PR DESCRIPTION
**Pull request by Nisheanthan**
- Added hi

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # PregRed
 Readiness Checker for Healthy Delivery
+Hi Nishean
```
Comment (Nisheanthan) [inline README.md:3]: don't capitalise first word

Comment (Nisheanthan) [reply README.md:3]: done

Comment (Chiran2018814) [inline README.md:3]: This is OK

**Review by UmeshikaMahaliyadda — CHANGES_REQUESTED** [review]
please dont do the same thing again

**Pull request merged**